tests/config/rector-phpunit.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Core\Configuration\Option;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+/**
+ * Assumes drupal/recommended-project is at tests/fixtures/drupal
+ *
+ * composer create-project drupal/recommended-project:^8.9 tests/fixtures/drupal
+ * composer create-project drupal/recommended-project:^9.0 tests/fixtures/drupal
+ */
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->import(__DIR__ .  '/../../config/drupal-8/drupal-8-all-deprecations.php');
+
+    $parameters = $containerConfigurator->parameters();
+    $parameters->set(Option::AUTOLOAD_PATHS, [
+        __DIR__ . '/../fixtures/drupal/web/core',
+        __DIR__ . '/../fixtures/drupal/web/core/modules',
+        __DIR__ . '/../fixtures/drupal/web/modules',
+        __DIR__ . '/../fixtures/drupal/web/profiles'
+    ]);
+    require_once __DIR__ . '/../fixtures/drupal/web/core/tests/bootstrap.php';
```

Comment:
This causes a bug, see

```
PHP Fatal error:  Cannot declare class Symfony\Component\DependencyInjection\Compiler\MergeExtensionConfigurationParameterBag, because the name is already in use in /home/runner/work/drupal-rector/drupal-rector/tests/fixtures/drupal/vendor/symfony/dependency-injection/Compiler/MergeExtensionConfigurationPass.php on line 135
```